GitUI/CommandsDialogs/FormBrowse.InitMenusAndToolbars.cs

```diff
@@ -343,6 +344,7 @@ private Brush UpdateCommitButtonAndGetBrush(IReadOnlyList<GitItemStatus>? status
                 toolStripButtonCommit.AutoSize = true;
             }
 
+            ToolStripMain.ResumeLayout();
```

Comment:
```suggestion
            finally
            {
                ToolStripMain.ResumeLayout();
            }
```

Reply:
done, but I do not like the change, null handling (to avoid allocate)

Reply:
me neither
IIRC, the `return brush` can be moved into the `try` block.

Reply:
null annotation was not necessary